internal/nix/shell.go

```diff
@@ -252,6 +282,32 @@ func (s *Shell) Run(nixShellFilePath string) error {
 	return errors.WithStack(err)
 }
 
+func (s *Shell) computeNixShellEnv(nixShellFilePath string) (map[string]string, error) {
+	vaf, err := PrintDevEnv(nixShellFilePath)
```

Comment:
what does `vaf` stand for?

Reply:
variables and functions, but that's really long...